test/debtArbitrator.js

```diff
@@ -24,7 +24,7 @@ contract('DebtArbitrator', function(accounts) {
       const value = 1000;
       return Promise.resolve()
       .then(() => debtArbitrator.borrow(value, {from: borrower}))
-      .then(() => debtArbitrator.debts(borrower))
+      .then(() => debtArbitrator.debts.call(borrower))
```

Comment:
As the `debts()` is marked as constant in the contract, `.call` is added implicitly. Though you can add it if you want.